AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/Statement+Model.swift

```diff
@@ -8,6 +8,7 @@
 import Amplify
 import Foundation
 import SQLite
+import AWSPluginsCore
 
 typealias ModelValues = [String: Any?]
 
```

Comment:
Below: a `public` func (`from(dictionary:)`) on a non-public type doesn't make sense. Remove `public` from the func